src/__integrations__/testCases/transientConfigurableFetch.ts

```diff
@@ -0,0 +1,13 @@
+import { fetch as polyfilledFetch } from 'whatwg-fetch';
+import { transientConfigurableFetch } from '../../utils/fetch';
+
+function transientConfigurableFetchTest() {
+  // Set error 404 as transient error.
+  const myFetch = transientConfigurableFetch(polyfilledFetch, 3, 404);
```

Comment:
the interface of `transientConfigurableFetch` is somewhat clumsy, that would have been better to have a configurable object for params to provide meaningful names (what are `3` and `404` in this context ?)